FhirToDataLake/native/parquet/Nuget.Config

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Add this nuget.config for Native project restore nuget packages.
Multiple nuget settings will be applied in the path from the drive root to the current folder, see [how settings are applied](https://learn.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#how-settings-are-applied).